src/helm/proxy/tokenizers/huggingface_tokenizer.py

```diff
@@ -155,6 +155,9 @@ def _tokenize_do_it(self, request: Dict[str, Any]) -> Dict[str, Any]:
                 # Just like tokenize("Hello", encode=False) would return ["Hello"].
                 with self._get_tokenizer_for_request(request) as tokenizer:
                     tokens = tokenizer.tokenize(request["text"])
+                # Some tokenizers (e.g. Qwen/Qwen-7B) return the tokens as bytes, so we have to decode them to strings.
+                if tokens and type(tokens[0]) == bytes:
+                    tokens = [cast(bytes, token).decode(errors="ignore") for token in tokens]
```

Comment:
Do we want to ignore errors?  Might there be some silent failures?

Reply:
The alternatives is ['replace' or 'backslashreplace' or 'surrogateescape'](https://docs.python.org/3/library/codecs.html#error-handlers), which will insert sentinel string where the error occurred. Would you prefer one of those?

Tokenization into strings is best-effort anyway - there is no general guarantee across tokenizers that `"".join(tokens)` gives you the original string. For instance, a tokenization boundary can be in the middle of bytes of a single UTF-8 character.